server/server/common/arangodb.py

```diff
@@ -76,13 +76,13 @@ def __init__(self, app=None):
     def connect(self) -> ArangoClient:
         """Connect to the ArangoDB"""
         config = current_app.config['ARANGO_CLIENT']
-        return ArangoClient(host=config['host'], port=config['port'])
+        return ArangoClient(hosts=f'http://{config["host"]}:{config["port"]}')
 
     @property
     def client(self) -> ArangoClient:
         """
         Puts client object in g object.
-        :return: ArangoClient object connected to the databse.
+        :return: ArangoClient object connected to the database.
```

Comment:
The changed line here throws an error with `make delete-database`. There appears to be a typo in it, but even when that's corrected there's still an error.

Reply:
What kind of error? I just ran it and it's okay...